atlas/packages/app/tsconfig.json

```diff
@@ -2,8 +2,25 @@
 	"extends": "../../tsconfig.json",
 
 	"compilerOptions": {
+		"baseUrl": "./",
 		"declaration": true,
 		"declarationDir": "dist",
-		"lib": ["DOM", "ES2019"]
-	}
+		"lib": ["DOM", "ES2019"],
+		"paths": {
+			"components": ["src/components"],
```

Comment:
I think it would be clearer to prefix paths with `@` or `~` to indicate that they come from the same repo and not from an outside package

Reply:
Like we've talked, using `@` as a root was the first thing I wanted to do. However, Parcel allows neither `@` and `~` in the alias, as it reserves those for some specific use cases. That really sucks though, Webpack allows both :/

Reply:
So I guess we could use some other character (`#`?) for the root or just leave those be as they are. To be honest I don't like either of those solutions that much but it's still better than doing relative paths everywhere. We can also switch to Webpack haha

 What do you think?

Reply:
Let's leave it as is for now, and tackle a possible migration to webpack in a next PR if we start to see the benefits of doing so

Reply:
:+1: